### ISSUE TITLE
No user name when registering.

### ISSUE THREAD

**Issue by xBrownieCodez** [ThalKod/discord-clone#127]
You can't choose a username while registering.

**Comment by ThalKod** [comment]
You signup with your mail, You can then choose a username....

**Comment by xBrownieCodez** [comment]
Can you add a Username selection when registering like on Discord?

**Comment by unknownguy2002** [comment]
@ThalKod Open for a PR about this?

**Comment by ThalKod** [comment]
@unknownguy2002 Yes, go ahead :) while at it, you should also add email and username validation in the form, basically create routes and handle this via ajax, on input blur...

**Comment by unknownguy2002** [comment]
I'm not too free this week, but I'll slowly add these features in. Also, do you need a test server to show this?

**Comment by ThalKod** [comment]
OK, no problem.

**Comment by eklavya11** [comment]
can u show us a example by glitch.com ?
that may give us a guide .

**Comment by jamesrchen** [comment]
okay sorry for being away for a VERY VERY Long time. I doubt this will change.

However, I do have a server up hosting this project for fun. I might make an issue about it soon so we can add it in the readme.md.
